src/data/queries.yml

```diff
@@ -0,0 +1,18 @@
+# Keep all keys sorted (enforced via CI)
+queries:
+  java_io_exception_remote_call:
```

Comment:
why is there no "name" which is "required"?

Reply:
Well, until we add a schema to validate the file  I will rely on peer reviews to spot mistakes.